BranchSDK-Samples/Windows/ColorPicker/README.md

```diff
@@ -0,0 +1,77 @@
+Native C++ ColorPicker Sample
+===================================
+
+Branch Sample written for Windows that demonstrates
+
+* Branch SDK Integration
+* Branch Initialization
+* Responding to a deep link event
+
+Pre-requisites
+--------------
+
+- Visual Studio
+
+Getting Started
+---------------
+
+This sample uses Visual Studio. 
+
+To build this project, use the "Open Project" in Visual Studio and select the solution.
+
+
+Using the ColorPicker App
+-------------------------
+
+### Basic Usage
+
+Running the application out of the box will demonstrate Branch initialization.   
+Debug Output will show in the debugger when Branch responsds with the **Open** request.
+
+The window background is a nice shade of blue by default.
+
+Further, a Color Dialog will show when the application is first opened allowing you to
+pick a different background color.  Selecting a new color will send an event to Branch
+inidicating "TUTORIAL_COMPLETE".
+
+### Advanced Usage
+
+Clicking on a branch link prior to opening the application can show what happens when the application
```

Comment:
I'd remove "prior to opening the application" for clarity unless I'm missing something (i.e. deeplink doesn't work when the app is in the background)

Reply:
Gotcha!